lib/fastlane/mail_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Fastlane
+  class MailHelper
```

Comment:
I would maybe suggest remaining this to `TemplateHelper` and `template_helper.rb`. Looks a bit weird being `MailHelper` in the `s3` action :innocent: